utopian-io/v2.utopian.io
# Add collaborators to a project

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

---

### **Test plan**

_Create a project with collaborators_

* Fill the form with the required fields
* Add a collaborator

**Expected result:** You should see the collaborator added in the db with the selected roles

_Edit the collaborators of a project_

* Add a new collaborator
* Edit the roles of the existing one

**Expected result:** You should see the collaborators added in the db with the selected roles

_UI exclusion rules_

* Verify that an owner can't be added to the collaborators list
* Verify that a collaborator can't be added to the owners list
* Verifiy that the logged user can't be added to any of the lists
